tokamak/1-no-currents/BOUT.inp

```diff
@@ -0,0 +1,309 @@
+
+NOUT = 100
+TIMESTEP = 1000
+
+TwistShift = false  # use twist-shift condition?
+ballooning = true
+shiftinitial = false
+
+MZ = 1     # number of points in z direction (2^n + 1)
+
+# NXPE = 10
+
+ZPERIOD=1
+
+grid = tokamak-example.nc
+
+[restart]
+init_missing = true
+
+[mesh]
+paralleltransform = shifted
+
+symmetricGlobalX = true
+
+shiftangle_from_zshift=true
+
+extrapolate_y = false
+
+##################################################
+# derivative methods
+
+[ddx]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddy]
+
+first = C2
+second = C2
+upwind = W3
+
+[ddz]
+
+first = C2
+second = C2
+upwind = W3
+
+###################################################
+# Time-integration solver
+
+[solver]
+mxstep = 1000000
+
+# Note: If evolving neutrals, need preconditioning
+type = cvode
+use_precon = true
+
+maxl = 10
+
+ATOL = 1.0e-10  # absolute tolerance
+RTOL = 1.0e-5   # relative tolerance
+mxstep = 1000000  # Maximum internal steps per output
+
+[laplace]  # This is used for Delp2 operator
+all_terms = true
+nonuniform=true
+
+##################################################
+# Electrostatic potential solver
+
+[phiSolver]
+inner_boundary_flags = 0
```

Comment:
Shouldn't this be `INVERT_SET` too?

Reply:
Not sure: In the inversion we can have one boundary zero-gradient, just not both. It may work with INVERT_SET, but for now I've been using neumann on the inner boundary.